strings/names/sectoid_first.txt

```diff
@@ -0,0 +1 @@
+Sectoid
```

Comment:
This file is not necessary either, right?

Reply:
my bad, i forgot about those files